src/main/java/de/dentrassi/rpm/builder/RpmMojo.java

```diff
@@ -30,12 +30,7 @@
 import java.time.Instant;
 import java.time.ZoneOffset;
 import java.time.format.DateTimeFormatter;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
I would prefer to have expanded imports.

Reply:
Okay sure. Apologies, my IDE (IntelliJ) insists on doing this automated "improvements" for me.